azure-pipelines.yml

```diff
@@ -919,7 +919,7 @@ jobs:
 - job: VerifySphinxDocumentGenerator
   displayName: "Verify Sphinx Document Generator"
   pool:
-    vmImage: 'ubuntu-20.04'
+    name: 'pool-ubuntu-2004'
```

Comment:
Is it possible that we define a constant for pool name instead of hardcoded for easy maintenance?

Reply:
Add pool name to global variable template `.azure-pipelines/templates/variables.yml`